### PR TITLE
feat(git export): redirect to projects

### PR DESCRIPTION
**Pull request by danilowoz**
CodeSandbox logoCodeSandbox logo&nbsp; Open in CodeSandbox <a href="https://codesandbox.io/p/github/codesandbox/codesandbox-client/feat/redirect-git-to-projects?workspace=%7B%22gitSidebarPanel%22:%22PR%22,%22sidebarPanel%22:%22GIT%22%7D">Web Editor</a> | <a href="https://codesandbox.io/p/vscode?owner=codesandbox&repo=codesandbox-client&branch=feat/redirect-git-to-projects">VS Code</a>

Closes [CSB-5126](https://linear.app/codesandbox/issue/CSB-5091/implement-graphql-code-generator-on-web-projects)

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/6804/commits/dd36b2f36b3a1b6a768537ac9157f77a054dd599 is at https://pr6804.build.csb.dev/s/new.

File: packages/common/src/utils/url-generator.ts [hunk 1]
```diff
@@ -11,6 +11,20 @@ const sandboxHost = {
...
+  repo,
+}: {
+  owner: string;
+  repo;
+}) => {
+  const origin = process.env.STAGING_API
+    ? 'https://codesandbox.stream'
+    : 'https://codesandbox.io';
+
+  return `${origin}/p/${owner}/${repo}?create=true`;
```
Comment (tromika) [inline packages/common/src/utils/url-generator.ts:25]: `${origin}/p/${owner}/${repo}?create=true` -> `${origin}/p/github/${owner}/${repo}?create=true`

Comment (tromika) [inline packages/common/src/utils/url-generator.ts:25]:
```suggestion
  return `${origin}/p/github/${owner}/${repo}?create=true`;
```

**Review by tromika — APPROVED** [review]
LGTM!

**Pull request merged**